Sources/Meta/Visitors/CopyVisitor.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftSyntax
+
+public class CopyVisitor: SyntaxVisitor {
+    public private(set) var generatedCode = ""
+    
+    override public func visit(_ node: StructDeclSyntax) -> SyntaxVisitorContinueKind {
+        let structName = node.identifier.description.trimmingCharacters(in: .whitespacesAndNewlines)
+        let fields = node.fields
+        
+        let generated = """
+                        extension \(structName) {
+                            func copy(\(copyParameters(for: fields))) -> \(structName) {
+                                \(body(for: structName, with: fields))
+                            }
+                        }
+                        
+                        """
+        print(generated, to: &generatedCode)
+        
+        return .skipChildren
+    }
+    
+    private func copyParameters(for members: [Field]) -> String {
+        members.map { member in
+            "with\(member.name.capitalized) \(member.name): \(member.type)? = nil"
+        }.joined(separator: ",\n\t\t\t  ")
+    }
+    
+    private func body(for structName: String, with members: [Field]) -> String {
```

Comment:
maybe another name helps to understand better the behaviour of this method, something like `constructor(for:withFields:)` or similar

Reply:
The function is creating the `body` of the copy method, so I'd maintain this name.